src/frontend/pages/game.tsx

```diff
@@ -29,25 +32,69 @@ export default function Home(): ReactElement {
    * playerCoins: total # of coins that this user has in this game
    *
    */
+  const [pointsRemaining, setPointsRemaining] = useState<number>(
+    gameConstants.INIT_TOTAL_COINS,
+  );
   const [takeVal, setTakeVal] = useState<number>(gameConstants.MIN_TAKE_VAL);
   const TIMER_SECONDS = 10;
   const [modalIsOpen, setModalIsOpen] = useState<boolean>(false);
   const [playerColor, setPlayerColor] = useState<string>(
     gameConstants.DEFAULT_COLOR,
   );
-  const [playerCoins, setPlayerCoins] = useState<number>(
+  const [playerPoints, setPlayerPoints] = useState<number>(
     gameConstants.INIT_PLAYER_COINS,
   );
+  const [roundNumber, setRoundNumber] = useState<number>(1);
+  const router = useRouter();
+  const { gameId, playerId } = router.query;
+  const [timeLeft, setTimeLeft] = useState<number>(
+    gameConstants.INIT_TIME_LEFT,
+  );
+  const [takeComplete, setTakeComplete] = useState<boolean>(false);
+  const [gameOverModalIsOpen, setGameOverModalIsOpen] = useState<boolean>(
+    false,
+  );
+  const [waitModalIsOpen, setWaitModalIsOpen] = useState<boolean>(false);
 
-  const handleTake = (event) => {
-    event.preventDefault();
-    alert(`you took ${takeVal} coins!`);
+  /* TODO:
+  - Points remaining is wrong
+  */
```

Comment:
Is this todo still accurate?

Reply:
Did not manual test thoroughly. Count is not decrementing when taking 0.

Reply:
I tested, forgot to remove this todo

Reply:
we can just remove then